incubator/hnc/internal/reconcilers/object.go

```diff
@@ -382,6 +388,24 @@ func (r *ObjectReconciler) syncPropagated(ctx context.Context, log logr.Logger,
 	return actionNop, nil
 }
 
+func (r *ObjectReconciler) getSelector(ctx context.Context, log logr.Logger, inst *unstructured.Unstructured) labels.Selector {
+	annot := inst.GetAnnotations()
+	selectorstr, present := annot[selectorAnnot]
+	if present == false {
+		return nil
+	}
+	labelSelector, err := v1.ParseToLabelSelector(selectorstr)
+	if err != nil {
+		log.Error(err, "Could not parse selector annotation to labelSelector")
+		return nil
+	}
+	selector, err2 := v1.LabelSelectorAsSelector(labelSelector)
```

Comment:
No need to introduce a new name - `err` should work fine.

Reply:
done